web/src/filter.ts

```diff
@@ -562,9 +575,9 @@ export class Filter {
 
         if (terms.length >= 2) {
             const is = (term: NarrowTerm, expected: string): boolean =>
-                term.operator === expected && !term.negated;
+                util.canonicalize_stream_synonym(term.operator) === expected && !term.negated;
```

Comment:
This could probably safely be `self.canonicalize_operator` as a nicer abstraction.

Reply:
Updated to use `Filter.canonicalize_operator` here.